native/src/main/resources/META-INF/native-image/io.ballerina.stdlib/tcp-native/reflect-config.json

```diff
@@ -0,0 +1,144 @@
+      {"name":"deregister","parameterTypes":["io.netty.channel.ChannelHandlerContext","io.netty.channel.ChannelPromise"] },
+      {"name":"disconnect","parameterTypes":["io.netty.channel.ChannelHandlerContext","io.netty.channel.ChannelPromise"] },
+      {"name":"exceptionCaught","parameterTypes":["io.netty.channel.ChannelHandlerContext","java.lang.Throwable"] },
+      {"name":"flush","parameterTypes":["io.netty.channel.ChannelHandlerContext"] },
+      {"name":"read","parameterTypes":["io.netty.channel.ChannelHandlerContext"] },
+      {"name":"userEventTriggered","parameterTypes":["io.netty.channel.ChannelHandlerContext","java.lang.Object"] },
+      {"name":"write","parameterTypes":["io.netty.channel.ChannelHandlerContext","java.lang.Object","io.netty.channel.ChannelPromise"] }
+    ]
+  },
+  {
+    "name":"io.netty.channel.DefaultChannelPipeline$TailContext",
+    "queriedMethods":[
+      {"name":"channelActive","parameterTypes":["io.netty.channel.ChannelHandlerContext"] },
+      {"name":"channelInactive","parameterTypes":["io.netty.channel.ChannelHandlerContext"] },
+      {"name":"channelRead","parameterTypes":["io.netty.channel.ChannelHandlerContext","java.lang.Object"] },
+      {"name":"channelReadComplete","parameterTypes":["io.netty.channel.ChannelHandlerContext"] },
+      {"name":"channelRegistered","parameterTypes":["io.netty.channel.ChannelHandlerContext"] },
+      {"name":"channelUnregistered","parameterTypes":["io.netty.channel.ChannelHandlerContext"] },
+      {"name":"channelWritabilityChanged","parameterTypes":["io.netty.channel.ChannelHandlerContext"] },
+      {"name":"exceptionCaught","parameterTypes":["io.netty.channel.ChannelHandlerContext","java.lang.Throwable"] },
+      {"name":"userEventTriggered","parameterTypes":["io.netty.channel.ChannelHandlerContext","java.lang.Object"] }
+    ]
+  },
+  {
+    "name":"io.netty.handler.timeout.IdleStateHandler",
+    "queriedMethods":[
+      {"name":"channelActive","parameterTypes":["io.netty.channel.ChannelHandlerContext"] },
+      {"name":"channelInactive","parameterTypes":["io.netty.channel.ChannelHandlerContext"] },
+      {"name":"channelRead","parameterTypes":["io.netty.channel.ChannelHandlerContext","java.lang.Object"] },
+      {"name":"channelReadComplete","parameterTypes":["io.netty.channel.ChannelHandlerContext"] },
+      {"name":"channelRegistered","parameterTypes":["io.netty.channel.ChannelHandlerContext"] },
+      {"name":"write","parameterTypes":["io.netty.channel.ChannelHandlerContext","java.lang.Object","io.netty.channel.ChannelPromise"] }
+    ]
+  },
+  {
+    "name":"io.netty.channel.SimpleChannelInboundHandler",
+    "queriedMethods":[{"name":"channelRead","parameterTypes":["io.netty.channel.ChannelHandlerContext","java.lang.Object"] }]
+  },
+  {
+    "name":"io.netty.channel.socket.nio.NioServerSocketChannel",
```

Comment:
Can we try to remove this and test? This class is already added by [netty:transport](https://github.com/netty/netty/blob/4.1/transport/src/main/resources/META-INF/native-image/io.netty/transport/reflection-config.json).